source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -444,6 +448,8 @@ def _getColumnLocationRaw(self, index: int) -> Optional[RectLTRB]:
 		return RectLTRB(left, top, right, bottom).toScreen(self.windowHandle).toLTWH()
 
 	def _getColumnLocation(self, column: int) -> Optional[RectLTRB]:
+		if not self.parent._columnOrderArray:

```

Comment:
What was the falsey value that was not None?
It seems like the implementation of _get__columnOrderArray` doesn't match its type annotation.

Reply:
A bug was exposed here when testing StartIsBack, before the patch. The code was indexing NoneTypes with `self.parent._columnOrderArray[column - 1]` after the fetch failed. My earlier changes made `_get__columnOrderArray` no longer return None, as the Out-Of-Process fetch returned an empty array on a failed fetch.

I've updated the code so that the Out-Of-Process fetch returns None when the SendMessage fails, instead of a falsey value, to be consistent with the in-process fetch.

I also noticed that it appears the result was being handled incorrectly, as successful fetches were logging the debug warning when testing this patch.